tools/ci/get-ci-matrix.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
```

Comment:
you should be able to make a `util.py` to share utilities between the different scripts